content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -176,7 +176,7 @@ securityContext:
   fsGroupChangePolicy: "OnRootMismatch"
 ```
 
-This is an alpha feature. To use it, enable the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) `ConfigurableFSGroupPolicy` for the kube-api-server, the kube-controller-manager, and for the kubelet.
+This is feature is disabled by default in Kubernetes versions <= v1.19. To use it, enable the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) `ConfigurableFSGroupPolicy` for the kube-api-server, the kube-controller-manager, and for the kubelet.
```

Comment:
This documentation is for the latest Kubernetes version, and we should word things assuming that the reader has picked the right version. We should also help users decide which docs version to read.

I recommend using a [feature-state shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state) to highlight that docs for older versions might differ. See line 152.

Because that shortcode exists, we can assume that the reader has seen the feature state call out. Our job then is to explain that `ConfigurableFSGroupPolicy` is enabled by default.

Reply:
@sftim I updated the PR to try to adhere to your notes.

I tried to match the phrasing about "enabled by default" from https://kubernetes.io/docs/concepts/storage/persistent-volumes/#expanding-persistent-volumes-claims